chain/cosmos/chain_node.go

```diff
@@ -922,12 +922,12 @@ func (tn *ChainNode) ExportState(ctx context.Context, height int64) (string, err
 	tn.lock.Lock()
 	defer tn.lock.Unlock()
 
-	_, stderr, err := tn.ExecBin(ctx, "export", "--height", fmt.Sprint(height))
+	stdout, _, err := tn.ExecBin(ctx, "export", "--height", fmt.Sprint(height))
 	if err != nil {
 		return "", err
 	}
-	// output comes to stderr for some reason
-	return string(stderr), nil
+
+	return string(stdout), nil
```

Comment:
Can we concatenate stdout and stderr so that it is backwards compatible with prior binaries where the state export comes to stderr? We made this change in the v3 branch already but looks like it was not brought forward into main.

https://github.com/strangelove-ventures/interchaintest/blob/v3/chain/cosmos/chain_node.go#L879-L889

Reply:
Updated, thank @agouin